source/advanced_components/ddc/reference/provision/overview.rst

```diff
@@ -163,10 +165,10 @@ This section provides parameters for the host configuration process (e.g. KVM in
 
 .. _ddc_provision_template_devices:
 
-Provision objects
------------------
+OpenNebula infrastructure objects
```

Comment:
Have a similar simplified section for "virtual objects" at the end of this page.